torch/fx/graph.py

```diff
@@ -131,19 +135,25 @@ def _name(self, target: Target) -> str:
     def python_code(self, root_module: str) -> Tuple[str, str, List[str]]:
         free_vars: List[str] = []
         body: List[str] = []
+
+        def insert_hierarchy_guard(node : Node):
```

Comment:
it'd be nice to group nodes with the same qualname together in order to have fever `with` (and it'd be more readable too). Feel free to treat it as a future improvement suggestion

Reply:
Done!